pkg/pulsar/utils/function_confg.go

```diff
@@ -68,6 +68,8 @@ type FunctionConfig struct {
 	// A generalized way of specifying inputs
 	InputSpecs map[string]ConsumerConfig `json:"inputSpecs,omitempty" yaml:"inputSpecs"`
 
+	CustomRuntimeOptions string `json:"customRuntimeOptions,omitempty" yaml:"customRuntimeOptions"`
```

Comment:
We should keep the same name with https://github.com/streamnative/pulsarctl/blob/master/pkg/pulsar/utils/source_config.go#L44

Reply:
hey there @nodece thanks for the answer!
i've based it on the api docs from here https://pulsar.apache.org/sink-rest-api/?version=2.8.0#operation/registerSink
it seems that there is `runtimeFlags` and `customRuntimeOptions` as far as I understood from [here](https://pulsar.apache.org/docs/en/functions-runtime/) the latter is used when you have to customize things on the k8s pod.
are the fields related? the docs are not clear (at least on functions) what `runtimeFlags` mean.
let me know what to do, if you want i can also add both of them :)

Reply:
You are right.